client/src/components/pages/Home/index.js

```diff
@@ -27,6 +28,18 @@ class Home extends Component {
 
   changeSearch = ({ target: { value } }) => {
     this.setState({ search: value });
+
+    api
```

Comment:
why you are calling the `userNativeLang`  again when we are using it on the `componentDidMount`
so we have the data that we will filter it  !!